### DISCUSSION TITLE
post/2/

### DISCUSSION THREAD

**Discussion by giscus** [HoshinoRei/blog#3] [General]
# post/2/

小玲以前在使用 Windows 系统时，一直都是用系统自带的输入法——微软拼音的，而且小玲没有那种安装第三方输入法的习惯。但是有一天，小玲在网上看到好多人推荐的 Rime 输入法，抱着试一下的想法，小玲试用了一下这款输入法。没想到这款输入法的可定制性真的高。小玲从此就离不开这款输入法了。现在小玲也来向大家传教。

https://blog.bling.moe/post/2/

**Comment by QAQ-Huaxia** [comment]
嘿，这个组合键切换中英文的好用，终于找到了，感谢

**Comment by ace-3721** [comment]
终于可以愉快的在小狼毫上使用 `Ctrl+Space` 了。能否提供一份恢复系统默认的注册表代码呢？

**Reply by QAQ-Huaxia** [reply]
我把整个Hot Keys项都备份了
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys]

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000010]
"Key Modifiers"=hex:02,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:20,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000011]
"Key Modifiers"=hex:04,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:20,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000012]
"Key Modifiers"=hex:02,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:be,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000070]
"Key Modifiers"=hex:02,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:20,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000071]
"Key Modifiers"=hex:04,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:20,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000072]
"Key Modifiers"=hex:03,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:bc,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000200]
"Key Modifiers"=hex:03,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:47,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000201]
"Key Modifiers"=hex:03,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:4b,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000202]
"Key Modifiers"=hex:03,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:4c,00,00,00

[HKEY_CURRENT_USER\Control Panel\Input Method\Hot Keys\00000203]
"Key Modifiers"=hex:03,c0,00,00
"Target IME"=hex:00,00,00,00
"Virtual Key"=hex:56,00,00,00

**Comment by QAQ-Huaxia** [comment]
不过可惜的是我使用博主的输入法禁用的注册表方法貌似不生效，issues上的可以解决，那个快捷键貌似也有点问题不知道是不是我系统环境的问题，附上官方issues的方法https://github.com/rime/home/issues/803

**Comment by ace-3721** [comment]
很诡异的是，下班回家之后再开机，ctrl-space依然变成输入法被禁用，哪怕重新导入那个reg。
系统是WIN 11

**Reply by QAQ-Huaxia** [reply]
用这个试试https://github.com/rime/home/issues/803

**Reply by QAQ-Huaxia** [reply]
那个issues里面图片的方法有效，对我而言

**Reply by ace-3721** [reply]
我大概跟踪到了问题，确认解决之后和大家分享
